src/web/backend/nextcloud/serializers.py

```diff
@@ -0,0 +1,45 @@
+from marshmallow import Schema, fields, pre_load, post_load, pre_dump
+
+from .models import User
+
+
+class UserSchema(Schema):
+    """ Receive objects from edap library, serialize to user object class """
+    fqdn = fields.Str()
+    uid = fields.Str()
+    given_name = fields.Str(load_from='givenName', dump_to='givenName')
+    mail = fields.Str()
+    object_class = fields.List(fields.Str)
```

Comment:
`object_class` is an implementation detail of how is a user stored in LDAP. In other words, we don't have to store this information with the object, as the app should know this through the means of asking EDAP or configuration.